components/SSI/DIDxWallet/Wallet/DIDOperations/SocialRecovery/index.tsx

```diff
@@ -263,6 +264,52 @@ function Component() {
     }
   };
 
+  const resolveDid = async (_username: string) => {
+    if (isValidUsername(_username)) {
+      await fetchAddr({ net, _username, _domain: "did" })
```

Comment:
With this condition is sufficient. It means the username is an existing NFT. So let's just do a fetch address and throw the error message when the name is nonexistent so the user knows that's not a valid guardian name. Thanks! @DevPH-Wistkey

Reply:
Done, thanks!